cmds/minigames/connect4.js

```diff
@@ -1,84 +1,141 @@
+                    inline: true,
+                };
+                // message send to red player
+                let rmsg = await msg.author.send(
+                    client.EmbedMaker(
+                        msg,
+                        game.stringify(),
+                        client.colors.red,
+                        [field1]
+                    )
+                );
+                // message send to yellow player
+                let ymsg = await iuser.send(
+                    client.EmbedMaker(
+                        msg,
+                        game.stringify(),
+                        client.colors.red,
+                        [field1]
+                    )
+                );
+
+                // react with emojis in reactions array
+                // using for...of loop
+                for (emoji of game.reactions) {
+                    await rmsg.react(emoji);
+                    await ymsg.react(emoji);
                 }
 
-                const rfilter = (reaction,user) => {
-                    let pass = game.reactions.includes(reaction.emoji.name) && !user.bot && user.id === game.redid
-                    return pass
-                }
-                const yfilter = (reaction,user) => {
-                    let pass = game.reactions.includes(reaction.emoji.name) && !user.bot && user.id === game.yellowid
-                    return pass
+                // since the bot is sending embed in DM, you don't need to check for user id
+                const filter = (reaction, user) => {
+                    let pass = game.reactions.includes(reaction.emoji.name) && !user.bot;
+                    return pass;
                 }
 
-                const redcollector = rmsg.createReactionCollector(rfilter,{ idle: 300*1000 })
-                const yellowcollector = ymsg.createReactionCollector(yfilter,{ idle: 300*1000 })
-
-                redcollector.on("collect", async (reaction,user) =>{
-                
-                    if(reaction.emoji.name === "🛑"){
-
-                        rmsg.edit(client.EmbedMaker(msg, client.language.commands.game[userlang].abrupted.replace("{0}",user.username), client.colors.black))
-                        ymsg.edit(client.EmbedMaker(msg, client.language.commands.game[userlang].abrupted.replace("{0}",user.username), client.colors.black))
-                        end = true
-                        redcollector.stop()
-                        yellowcollector.stop()
-                        return
-
+                const redcollector = rmsg.createReactionCollector(filter, {
+                    idle: 300 * 1000,
+                });
+                const yellowcollector = ymsg.createReactionCollector(filter, {
+                    idle: 300 * 1000,
+                });
+
+                // merge 2 collector if possible
+                redcollector.on("collect", async (reaction, user) => {
```

Comment:
the two collectors are essentially doing the same thing, consider using a function to merge the two collectors